pages/common/kubectl-rollout.md

```diff
@@ -0,0 +1,20 @@
+# kubectl rollout
+
+> Manage the rollout of a resource (deployments, daemonsets, and statefulsets).
```

Comment:
Would be great if you'd be a bit more specific here. Is this correct?
```suggestion
> Manage the rollout of a kubectl resource (deployments, daemonsets, and statefulsets).
```

Reply:
I think `Kubernetes` is more accurate here. I can make the change if you think that sounds better.
```suggestion
> Manage the rollout of a Kubernetes resource (deployments, daemonsets, and statefulsets).
```

Reply:
Sounds good